static/app/components/events/eventTags/eventTagsTree.tsx

```diff
@@ -282,16 +401,32 @@ const TreeKeyTrunk = styled('div')<{spacerCount: number}>`
 
 const TreeValueTrunk = styled('div')`
   grid-column: 2 / 3;
+  display: grid;
+  grid-template-columns: 1fr 25px;
+  grid-column-gap: ${space(0.5)};
 `;
 
-const TreeValue = styled('span')`
+const TreeValue = styled('div')`
   font-family: ${p => p.theme.text.familyMono};
   word-break: break-word;
+  grid-column: span 1;
 `;
 
 const TreeKey = styled(TreeValue)`
-  grid-column: span 1;
   color: ${p => p.theme.gray300};
 `;
 
+const TreeValueDropdown = styled(DropdownMenu)`
+  justify-self: end;
+  margin: 1px;
+  height: 20px;
+  .tag-button {
+    height: 20px;
+    min-height: 20px;
+    padding: ${space(0)} ${space(0.75)};
+    border-radius: ${space(0.5)};
+    z-index: 0;
```

Comment:
Override the `z-index: 1` on tag buttons to ensure they live below underneath the parent `ClippedBox`.